### ISSUE TITLE
Travis CI - service running fails with no java

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/tomcat-formula#100]
States work on CentOS/Vagrant but fails in Travis CI.  **NOTE: Tomcat has dependency on Java!!**
```
----------
          ID: tomcat package installed and service running
    Function: service.running
        Name: tomcat
      Result: True
     Comment: Service tomcat has been enabled, and is running
     Started: 09:52:39.541845
    Duration: 273.18 ms
     Changes:   
              ----------
              tomcat:
                  True

[root@selenium-node salt]# systemctl list-unit-files | grep tomcat
tomcat.service                                enabled 
tomcat@.service                               disabled
```

Best solution might be to set `service_enabled: False` in pillar.example
 so Travis/Kitchen tests are not misleadingly failing.
```
                 ID: whats wrong with tomcat service
           Function: cmd.run
               Name: systemctl status tomcat8
             Result: False
            Comment: Command "systemctl status tomcat8" run
            Started: 18:57:55.444799
           Duration: 22.749 ms
            Changes:   
              ----------
              pid:
                  6412
              retcode:
                  3
              stderr:
              stdout:
                  ● tomcat8.service - LSB: Start Tomcat.
                     Loaded: loaded (/etc/init.d/tomcat8; generated)
                     Active: failed (Result: exit-code) since Fri 2019-02-15 18:57:50 UTC; 4s ago
                       Docs: man:systemd-sysv-generator(8)
                  
                  Feb 15 18:57:50 salt-formula.ci.local systemd[1]: Starting LSB: Start Tomcat....
                  Feb 15 18:57:50 salt-formula.ci.local systemd[1]: tomcat8.service: Control process exited, code=exited status=1
                  Feb 15 18:57:50 salt-formula.ci.local systemd[1]: tomcat8.service: Failed with result 'exit-code'.
                  Feb 15 18:57:50 salt-formula.ci.local systemd[1]: Failed to start LSB: Start Tomcat..
```

**Comment by noelmcloughlin** [comment]
One job has slightly different error - but probably same issue.

```
Setting environment variables from .travis.yml
$ export INSTANCE=extend-centos-7
... etc ...
       ----------
                 ID: tomcat package installed and service running
           Function: service.running
               Name: tomcat
             Result: False
            Comment: State 'service.running' was not found in SLS 'tomcat'
              Reason: 'service' __virtual__ returned False: No service execution module loaded: check support for service management on CentOS Linux-7
            Changes:  
```

**Comment by noelmcloughlin** [comment]
I tried setting `service_enabled: False` in pillar.example but Travis job did not detect this.

**Comment by noelmcloughlin** [comment]
I tried adding `service_running: False` in pillar.example but Travis job failed.

**Comment by noelmcloughlin** [comment]
If we set `service_running: False` in defaults.yaml the error probably goes away. Not an ideal default.

**Comment by myii** [comment]
Resolved by #110, with follow-up PRs #111 & #112.
